core/storage/question/gae_models_test.py

```diff
@@ -38,6 +39,25 @@ def test_create_question(self):
             question_model.question_state_data, question_state_data)
         self.assertEqual(question_model.language_code, language_code)
 
+    
+    def test_raise_exception_by_mocking_collision(self):
+        state = state_domain.State.create_default_state('ABC')
+        question_state_data = state.to_dict()
+        language_code = 'en'
+        version = 1
+        question_model = None
```

Comment:
Delete this.

Reply:
I think everything except `question_model` is needed since they are passed into `QuestionModel.create` function. But yes, I removed `question_model` since we no longer need it!